pages/index.tsx

```diff
@@ -117,32 +117,33 @@ const Frontpage: NextPage = () => {
                 <FormattedMessage id="frontpage.popularProductsHeader" />
               </Typography>
             </SectionHeader>
-
-            <GridContainer>
-              {popularDataProducts.length > 0 && popularDataProducts.map((product) => (
-                <Link key={product.id} href={{ pathname: "/assets/[id]", query: { id: product.id } }} title={product.name}>
-                  <CardGrid elevation="raised" onClick={() => {}}>
-                    <Card.Header>
-                      <AssetCardTitle>
-                        <Typography variant="h5" as="h2">
-                          {product.name}
+            {popularDataProducts.length > 0 ? (
```

Comment:
this makes the identical expression on line 122 redundant. suggest we just remove the check on line 122.

Reply:
True that!